### ISSUE TITLE
[Day19] Batch Normalization이 좋은 성능을 보이는 이유

### ISSUE THREAD

**Issue by nureesong** [Boost-Up-AI/Group-Activity#16]
## 📚 피어세션 자료 공유 
 
<img width="695" alt="batchnorm" src="https://user-images.githubusercontent.com/76163168/108344210-347c1880-7220-11eb-8498-b99c2be03e1e.png">
